server/form-pages/apply/risks-and-needs/health-needs/substanceMisuse.test.ts

```diff
@@ -13,6 +13,40 @@ describe('SubstanceMisuse', () => {
     })
   })
 
+  describe('Questions', () => {
+    const page = new SubstanceMisuse({}, application)
+
+    describe('usesIllegalSubstances', () => {
+      it('has a question', () => {
+        expect(page.questions.usesIllegalSubstances.question).toBeDefined()
```

Comment:
I'm afraid I instinctively don't really like the look of toBeDefined(). What do you think of exporting the questions variable from the substanceMisuse file, importing it into here and checking the values match?

Reply:
hmm, interesting point, but I'm not sure that asserting that:

```
expect(
  page.
    questions.usesIllegalSubstances.question).toEqual(
    importedQuestions.usesIllegalSubstances.question
  )
```

is any more meaningful?

My intention with this spec is to describe the structure of the questions: that there's one question asking whether illegal substances are being used, and within that 2 followup questions etc. It's deliberately not concerned with the type of question or the wording of the question as those are going to be covered in the description of the `response()`. 

If we document the precise wording of the questions in two places we will increase the maintenance burden -- with no benefit which I can see.

But maybe I am misunderstanding?

Reply:
ah yes, of course that's covered in the response() test. OK, I see what you're saying and I'm happy with that